File: Task6.1/app/src/main/java/com/example/task61/MainActivity.java
package com.example.task61;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import com.example.task61.fragment.ProfileFragment;
import com.example.task61.fragment.RecommendationFragment;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainActivity extends AppCompatActivity {

    BottomNavigationView bottomNavigationView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        bottomNavigationView = findViewById(R.id.bottomNavigationView);
        bottomNavigationView.setOnNavigationItemSelectedListener(item -> {
            Fragment selected = null;
            if (item.getItemId() == R.id.menu_home) {
                selected = new RecommendationFragment();
            } else if (item.getItemId() == R.id.menu_profile) {
                selected = new ProfileFragment();
            }
            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, selected).commit();
            return true;
        });

        bottomNavigationView.setSelectedItemId(R.id.menu_home);
    }
}

File: Task5.1C/app/src/main/java/com/example/itube/MainActivity.kt
package com.example.itube

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.example.itube.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.bottomNavigation.setOnItemSelectedListener { item ->
            when (item.itemId) {
                R.id.nav_news -> {
                    replaceFragment(NewsFragment())
                    true
                }
                R.id.nav_itube -> {
                    replaceFragment(HomeFragment())
                    true
                }
                R.id.nav_playlist -> {
                    replaceFragment(PlaylistFragment())
                    true
                }
                else -> false
            }
        }

        if (savedInstanceState == null) {
            replaceFragment(HomeFragment())
        }
    }

    private fun replaceFragment(fragment: Fragment) {
        supportFragmentManager.beginTransaction()
            .replace(R.id.fragmentContainer, fragment)
            .commit()
    }
} 